sd-finetuner-workflow/sd-finetune-workflow.yaml

```diff
@@ -0,0 +1,381 @@
+    # The directory to read your dataset in.
+    - name: dataset
+      value: 'dataset'
+    # The learning rate to use.
+    - name: lr
+      value: '5e-6'
+    # The amount of epochs to use.
+    - name: epochs
+      value: '10'
+    # Batch size
+    - name: batch_size
+      value: '1'
+    # Whether or not to use EMA (Exponential Moving Average) for the model during training.
+    - name: use_ema
+      value: 'False'
+    # Percentage chance of dropping out the text condition per batch. Ranges from 0.0 to 1.0 where 1.0 means 100% text condition dropout.
+    - name: ucg
+      value: '0.1'
+    # Whether or not to use Gradient Checkpointing. This is a memory optimization that trades off compute for memory.
+    - name: gradient_checkpointing
+      value: 'False'
+    # 8-bit Adam Optimizer - Only works on certain GPUs.
+    - name: use_8bit_adam
+      value: 'False'
+    # Adam beta 1
+    - name: adam_beta1
+      value: '0.9'
+    # Adam beta 2
+    - name: adam_beta2
+      value: '0.999'
+    # Adam weight decay
+    - name: adam_weight_decay
+      value: '1e-2'
+    # Adam epsilon
+    - name: adam_epsilon
+      value: '1e-8'
+    # Seed for reproducibility
+    - name: seed
+      value: '42'
+    # Checkpointing steps to save the model at.
+    - name: save_steps
+      value: '500'
+    # Image resolution to train at
+    - name: resolution
+      value: '512'
+    # Whether or not to resize the images during training.
+    - name: resize
+      value: 'False'
+    # Center crop the images during training.
+    - name: center_crop
+      value: 'False'
+    # Resizing interpolation method.
+    - name: resize_interp
+      value: 'lanczos'
+    # Whether or not to shuffle the dataset.
+    - name: shuffle
+      value: 'True'
+    # Image logging steps to log images at.
+    - name: image_log_steps
+      value: '500'
+    # Amount of images to log at every image logging step.
+    - name: image_log_amount
+      value: '4'
+    # Huggingface Token to download CompVis models.
+    - name: hf_token
+    # Wandb API key to report to wandb.
+    - name: wandb_api_key
+    # Project ID to report to wandb.
+    - name: project_id
+      value: 'sd-finetune'
+    # Inference service configuration.
+    - name: run_inference
+      value: 'False'
+    # Skip training and only run inference.
+    - name: inference_only
+      value: 'False'
+    # CoreWeave region to default to; ORD1 has most of the GPUs.
+    - name: region
+      value: 'ORD1'
+    # Training GPU - A6000, 48gb VRAM
+    - name: trainer_gpu
+      value: 'RTX_A6000'
+    # Inference GPU - Quadro RTX 5000, 16gb VRAM
+    - name: inference_gpu
+      value: 'Quadro_RTX_5000'
+    # Container images -- generally, don't alter this.
+    - name: downloader_image
+      value: 'ghcr.io/wbrown/gpt_bpe/model_downloader'
+    - name: downloader_tag
+      value: '797b903'
+    - name: finetuner_image
+      value: 'docker.io/harubaru1/sd-finetuner'
```

Comment:
Work with @todie to get an updated image pushed to the ML images repo, and update this.